content/docs/fundamentals/converting/resolving-errors@pt_BR.md

```diff
@@ -0,0 +1,350 @@
+---
+$title: Corrigir erros de validação
+$order: 2
+toc: true
+---
+
+[TOC]
+
+Nesta seção, analisaremos e resolveremos os erros de validação de AMP na nossa página AMP.  Esses erros podem aparecer em outra ordem no seu console.
+
+## Incluir charset
+
+Para começar, corrigiremos o seguinte erro:
+
+<pre class="error-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).